### DISCUSSION TITLE
SRW runs at sub-3km grid spacing

### DISCUSSION THREAD

**Discussion by hertneky** [ufs-community/ufs-srweather-app#634] [Q&A] [answered]
I understand that the SRW does not currently have capabilities for grids less than 3 km and therefore not supported. Just hoping to get some clarity on the status.

I know there has been work in the past from @dmwright526 [feature branch](https://github.com/ufs-community/UFS_UTILS/compare/develop...dmwright526:UFS_UTILS:feature/1kmGridRes), but that is quite outdated and was apparently never completed/merged to develop and the mods don't work with recent UFS_UTILS, such as in the v2.1 release (at least with my specific configuration). Note: Interestingly, if I use UFS_UTILS 1km feature branch as a whole rather than pulling the mods into the release code, it does work.....but as it's quite an old hash, it's not ideal to use with SRWv2.1.

1. What was the status of that feature and why did it never make it into develop, and are there plans to continue that effort?
2. Is there any other recent work being done to add sub-3km capabilities to SRW? (I have heard about convection-related issues with high res tests in the community and looking at MPAS instead, so maybe not...)

**Comment by ShawnCebulaNOAA** [comment]
Hello Tracy, hope you're doing well!

I'm not sure what the future plans are for the SRW so I'll check in with the team and see if they can let you know!

Shawn Cebula, EPIC End User Support

**Reply by hertneky** [reply]
Thanks @ShawnCebulaNOAA David was able to respond to my question about his 1km feature branch. At this point, my remaining question is if there is any other recent work in SRW for the 1-3 km range that I could leverage for testing.

**Comment by DavidWright-NOAA** [comment, marked as answer]
Hi @hertneky! That feature branch was never merged due to some of my own concerns in the use of this "solution" in all applications. For context, the current error occurs when creating a grid spacing of 1km or less (grid generation I believe should work between 1km and 3km). Generating a grid at 1km or less generates erroneous "water" points across the domain. This happens from the searching of land-use type to fill grid cells not being large enough and defaulting to land-use type of zero (water). All my branch really does is increase the search distance from +1 grid cell to +2. For the work I have done over the Great Lakes this is probably fine (not great), but I have concerns with potentially using this for something like warm-season convection which can be sensitive to land-use type/soil moisture. This work has since been put on the backburner for now due to a job transition on my end. I know this work was shared with CAPS (the reason for the feature branch exists) for testing on their end, but I am not aware if they have fully tested the UFS-SRW at this resolution yet.

**Reply by hertneky** [reply]
@DavidWright-NOAA Thank you very much for your reply. I had been pointed to your branch through a colleague who had given me the impression this was current ongoing work, before I had realized the feature was older. I am working with a domain at 1.8 km, so it is at least in the 1-3 km range. Grid generation is not an issue and seems ok, but there are model runtime issues with the more recent code. I will see if anyone else has additional information to add to this discussion related to any more recent work on sub-3km simulations.
